src/onegov/form/display.py

```diff
@@ -121,6 +119,71 @@ def __call__(self, field: 'Field') -> Markup:
         ).format(url=field.data)
 
 
+@registry.register_for('VideoURLField')
+class VideoURLFieldRenderer(BaseRenderer):
+    """
+    Renders a video url. Embeds the video if in case of vimeo or
+    youtube otherwise just displays the url as a link.
+    """
+
+    video_template = Markup("""
+        <div class="video">
+            <div class="videowrapper">
+                <iframe allow="fullscreen" frameborder="0" src="{url}"
+                sandbox="allow-scripts allow-same-origin
+                allow-presentation" referrerpolicy="no-referrer"></iframe>
+            </div>
+        </div>
+    """)
+    url_template = Markup('<a href="{url}">{url}</a>')
+
+    def __call__(self, field: 'Field', **kwargs: Any) -> Markup:
```

Comment:
```suggestion
    def __call__(self, field: 'Field') -> Markup:
```
You no longer need kwargs here